tests/python/relay/test_pass_simplify_expr.py

```diff
@@ -512,5 +512,34 @@ def test_concretize_multiple():
     assert tvm.ir.structural_equal(actual, expected)
 
 
+def test_simplify_consecuitive_add():
+    shape = (32, 1, 1)
+    c_data = np.empty(shape).astype("float32")
+    c1 = relay.const(c_data)
+    c2 = relay.const(c_data)
+
+    def before():
+        x = relay.var("x", shape=(1, 16, 16, 16), dtype="float32")
+        w = relay.var("w", shape=(32, 16, 3, 3), dtype="float32")
+        y = relay.nn.conv2d(x, w, padding=(1, 1))
+        y = relay.add(y, c1)
+        y = relay.add(y, c2)
```

Comment:
If const happens on left side, e.g.
```
 y = relay.add(c2, y)
```
Will this simplification work as well?

Reply:
Yes, the simplification works, no matter the const on the right or the left side. I have checked in the test_pass_smiplify_expr.py.

Reply:
Would you please add a test to cover this case?

Reply:
test case: const on the left is added in test_pass_smiplify_expr.py